src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -394,4 +394,10 @@
   <data name="InputFileNotSpecified" xml:space="preserve">
     <value>Please specify a nuspec, project.json, project file to use</value>
   </data>
+  <data name="Error_PackFailed" xml:space="preserve">
+    <value>Failed to build package.  The error message was '{0}'.</value>
```

Comment:
Can this just be:
Failed to build package. {0}

I don't know exactly what will go into {0}, so pick what makes the most sense on screen.